src/utils/integration_utils.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export function checkDialogElementForError(elem, value) {
+    if (!value && !elem.optional) {
+        return {
+            id: 'interactive_dialog.error.required',
```

Comment:
do we have this id's defined somewhere in the web and mobile apps? and the ones below? cause if we don't they are not going to get picked up by @jespino tool.

Also not sure if this localization stuff should be in redux, perhaps return a constant and use that constant in the apps?

Reply:
The same IDs will be used in both the webapp and RN. I don't really see a difference between returning a localization ID and creating a separate constant that would probably be the same thing

Reply:
Also the translations are already stored in redux state, right?

Reply:
sounds good, about the id instead of the constant, but the translations are NOT in the redux store, perhaps for the webapp but NOT in the mobile app for sure